pkg/controller/infrastructure/options.go

```diff
@@ -21,7 +21,7 @@ import (
 const (
 	// IgnoreOperationAnnotationFlag is the name of the command line flag to specify whether the operation annotation
 	// is ignored or not.
-	IgnoreOperationAnnotationFlag = "infrastructure-ignore-operation-annotation"
+	IgnoreOperationAnnotationFlag = "ignore-operation-annotation"
```

Comment:
Why this change? It's only relevant for `infrastructure` controller

Reply:
Yes but this should not be statically encoded into the flag name but rather be done by using the `PrefixOption` operator - for instance, if somebody ever wants to run an infrastructure controller on its own, then you probably wouldn't want to have this 'prefix'.

Reply:
OK, then let's use the `PrefixOption` operator.